developer_guides/debugability/logger/index.rst

```diff
@@ -194,6 +194,10 @@ A similar example may be prepared for components on a particular pipeline:
   To track verbose message use firmware build with selected "Trace verbose"
   option under "Trace" menu.
 
+Active trave filters are stored in firmware runtime memory, so after firmware
+restart (eg. after power gating in sleep mode) filters settings will be reset.
+Consider disabling power gating during your debug session.
```

Comment:
We should have the instruction for disabling runtime PM here. @plbossart @kv2019i is there a one line command that @ktrzcinx can insert here.

Reply:
@ktrzcinx I now see the debugfs filter is read only. Can we make it to be read/write and restore the filter when we send CTX_RESTORE?

Reply:
@lgirdwood I've added cmd to disable PM

@xiulipan I would be nice solution to restore filter setup, but there are a one problematic issue. FW doesn't store default filter settings, so to restore them, FW restart is needed, so in situation when logger is run-up twice with different filter setup without FW reboot, then combination of filters will be used - it already works like this, but now user must be aware of this, after introducing this automat, he won't be.

Reply:
@ktrzcinx Let's try to create a discussion about this feature update. I think end user will not use this, we can collect feedback from developer.